docs/themes/modifying_themes.md

```diff
@@ -41,4 +41,91 @@ The result in the default theme will look like::
                 - collection/
                     - browse.phtml
 ---
-```
\ No newline at end of file
+```
+
+## Theme Assets
+
+Non-view theme files live in the theme's `asset/` directory, organized into subdirectories by type. A typical `asset/` directory may look like the following:
+
+  
+```
+-   asset/
+    - css/
+    - sass/
+    - js/
+    - img/
+```
+
+  
+To refer to a file from the `assets/` directory within a view, use `$this->assetUrl('[subdirectory]/[filename.ext]')`. For example, if you wanted to display a theme's logo image within a view, it might look like this:
+
+`<img src=”<?php echo $this->assetUrl('img/logo.jpg'); ?>” title=”Logo”>`
+
+### Using Custom CSS and Javascript
+
+Every theme has its own version of layout.phtml, which provides the structure for each view. It typically includes:
+
+-   the header,
+-   the footer,
+-   the call for the page content (`<?php echo $this->content; ?>`),
+-   calls to fonts, css, and javascript files.
+
+Themes load their CSS and Javascript files using Zend's view helper functions ([`headLink()`](https://docs.zendframework.com/zend-view/helpers/head-link/) and [`headScript()`](https://docs.zendframework.com/zend-view/helpers/head-script/), which help control the order in which these files load alongside files from Omeka S's pages and  modules. These functions should appear before the `<head>` of layout.phtml.
+
+Here are examples for including these files in the <head> of a theme.
+
+For CSS:
+
+`$this->headLink()->prependStylesheet($this->assetUrl('css/style.css'));`
+
+The function `prependStylesheet()` will place the stylesheet at the top of the queue of loaded css files.
+
+`$this->headLink()->appendStylesheet($this->assetUrl('css/style.css'));`
+
+The function `appendStylesheet()` will place the stylesheet at the bottom of the queue of loaded CSS files.
+
+For Javascript:
+
+`$this->headScript()->prependFile($this->assetUrl('js/thedaily.js'));`
+
+The function `prependFile()` will place the stylesheet at the top of the queue of loaded javascript files.
+
+`$this->headScript()->appendFile($this->assetUrl('js/thedaily.js'));`
+
+The function `appendFile()` will place the javascript file at the top of the queue of loaded javascript files.
+
+### Using Webfonts
+
+Themers have access to a greater variety of fonts to use via hosted webfont libraries. Common examples include free resources like [Google Fonts](https://fonts.google.com) and [Adobe Edge Web Fonts](https://edgewebfonts.adobe.com/), as well as paid subscription sites such as [Fonts by Hoefler & Co](https://www.typography.com/webfonts/) and [Mosaic](https://www.monotype.com/fonts/mosaic). Hosted webfont usage often looks like the following:
+
+`<link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">`
```

Comment:
Since we're pushing the `//` protocol-relative-type url in the omeka code example we should probably use the same here.